# Generated by Django 2.1.1 on 2018-09-01 17:37

import django.contrib.auth.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('UsrMgmtModule', '0003_auto_20180901_1623'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='is_cook',
            field=models.BooleanField(blank=True, default=False),
        ),
        migrations.AlterField(
            model_name='user',
            name='last_name',
            field=models.CharField(blank=True, max_length=150, verbose_name='last name'),
        ),
        migrations.AlterField(
            model_name='user',
            name='phone_number',
            field=models.CharField(blank=True, max_length=10, validators=[django.core.validators.RegexValidator(message="Phone number must be entered in the format: '9999999999'.", regex='^\\d{10}$')]),
        ),
        migrations.AlterField(
            model_name='user',
            name='pincode',
            field=models.IntegerField(blank=True, default=500000, validators=[django.core.validators.RegexValidator(regex='^\\d{6}$')]),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
